### DISCUSSION TITLE
IBEX Image Analysis - QuPath+Cellpose, histoCAT or other ideas

### DISCUSSION THREAD

**Discussion by Katitamorais** [IBEXImagingCommunity/ibex_imaging_knowledge_base#130] [Q&A] [answered]
Hello!

Have you used QuPath+Cellpose or histoCAT for IBEX Image Analysis? We would to quantify individual markers as well as combination them similar to Flow Cytometry data.

If yes, could you share any recommendations for that?

Thanks.

**Comment by nadavyayon** [comment, marked as answer]
Hey! I have used cellpose to segment IBEX. It works well on both 2D and 3D. The parameters depend on the imaging conditions you have in your setup. I would recommend trying both nuclei and cyto models and test which works best. cyto tends to find larger masks around the nuclei which might be useful depending on your needs. 

Happy to provide more details. 

Nadav

**Comment by MeganVierhout** [comment]
Hi there! We have also done this but using the HALO image analysis platform (highplex module). We defined each cell population we wanted to quantify using a phenotype characterized by the various markers in our panels. We haven't tried Cellpose or histoCAT but will let you know if we do. 

Happy to chat more as well!

Megan

**Comment by radtkea** [comment]
Hi Katitamorais,

Great question! Thank you Nadav and Megan for your feedback.

In our original _PNAS_ paper we used HistoCat, but the developers have rapidly expanded their offerings: https://mcmicro.org/
Check out their latest updates. It's a rapidly evolving field.

I know several people who are quite happy with QuPath. This seems like a great first, open and free option.

This recent [paper](https://www.nature.com/articles/s41592-024-02233-6) on a cell segmentation challenge might be interesting for you.

I know a lot of people using HALO like Megan. I believe the software can be quick expensive but perhaps you could obtain an institutional discount?

Hope this helps!
A

**Comment by Katitamorais** [comment]
@radtkea, @MeganVierhout, and @nadavyayon, thank you for your reply.

Using QuPath we can run a reasonable pipeline analysis, for example, we can measure CD8+ positive cells and MAIT cells. In other words, quantification of single markers or creation of a combination of multiple markers for definition of classifiers in QuPath. However, we would like to have options for more sophisticated analysis such as spatial plots, histograms, tSNE and export results in flow cytometry format. I will get trial for HALO and that one in the link sent by Andrea.

There are some people also developing tools with machine learning.

If I have any news, I will let you know here!

Thanks.
